media/css/firefox/family/index-b.less

```diff
@@ -0,0 +1,305 @@
+    background-color: @blue;
+}
+
+.fxfamilynav-header {
+    // Safari 5.1+, Chrome 10+
+    background-image: -webkit-linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0)),
+        -webkit-linear-gradient(@blue, @blue);
+    background-image: -o-linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0)),
+        -o-linear-gradient(@blue, @blue);
+    // the standard
+    background-image: linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0)),
+        linear-gradient(@blue, @blue);
+}
+
+.content-section {
+    color: #484848;
+    padding: 100px 0;
+
+    @media only screen and (max-width: @breakTablet) {
+        padding: 50px 0;
+        text-align: center;
+    }
+}
+
+.content {
+    .clearfix();
+    background-repeat: no-repeat;
+    padding-left: 20px;
+    padding-right: 20px;
+    margin: 0 auto;
+    width: @widthDesktop - 40px;
+
+    a {
+        color: @blue;
+        font-weight: bold;
+    }
+
+    @media only screen and (max-width: @breakDesktop) {
+        width: @widthTablet - 40px;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        background-position: bottom 20px center;
+        width: @widthMobileLandscape - 40px;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding-left: 10px;
+        padding-right: 10px;
+        width: @widthMobile - 20px;
+    }
+}
+
+h2 {
+    .font-size(52px);
+    line-height: 1.3;
+    margin-bottom: 40px;
+    text-shadow: none;
+
+    @media only screen and (max-width: @breakDesktop) {
+        .font-size(46px);
+        margin-bottom: 34px;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        .font-size(38px);
+        margin-bottom: 30px;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        .font-size(28px);
+    }
+}
+
+h3 {
+    .font-size(36px);
+    line-height: 1.4;
+    margin-bottom: 40px;
+    text-shadow: none;
+
+    @media only screen and (max-width: @breakDesktop) {
+        .font-size(30px);
+        line-height: 1.3;
+        margin-bottom: 25px;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        .font-size(24px);
+    }
+}
+
+.copy {
+    width: 500px;
+
+    p {
+        .font-size(18px);
+        .open-sans-light();
+        line-height: 40px;
+    }
+
+    @media only screen and (max-width: @breakDesktop) {
+        width: 380px;
+
+        p {
+            .font-size(16px);
+            line-height: 34px;
+        }
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        width: auto;
+
+        p {
+            .font-size(14px);
+            line-height: 30px;
+        }
+    }
+}
+
+.fx-privacy-link {
+    a:active,
+    a:focus,
+    a:hover,
+    a:link,
+    a:visited {
+        color: #9b9b9b;
+    }
+}
+
+#intro {
+    &.content-section {
+        padding-top: 60px;
+    }
+
+    .content {
+        .at2x('/media/img/firefox/family/index-b/intro.png', 632px, 550px);
```

Comment:
Are you daring me to put a [Dimension](http://www.desktop-driver.com/wp-content/uploads/2013/02/Dell-Dimension-4100-450x336.jpg) in there instead? Because I will.